x/staking/keeper/cons_pubkey.go

```diff
@@ -105,37 +105,36 @@ func (k Keeper) updateToNewPubkey(ctx context.Context, val types.Validator, oldP
 	}
 
 	// sets a map: oldConsKey -> newConsKey
-	if err := k.OldToNewConsKeyMap.Set(ctx, oldPk.Address(), newPk.Address()); err != nil {
+	if err := k.OldToNewConsAddrMap.Set(ctx, oldPk.Address(), newPk.Address()); err != nil {
 		return err
 	}
 
 	// sets a map: newConsKey -> initialConsKey
-	if err := k.setConsKeyToValidatorIdentifierMap(ctx, sdk.ConsAddress(oldPk.Address()), sdk.ConsAddress(newPk.Address())); err != nil {
+	if err := k.setConsAddrToValidatorIdentifierMap(ctx, sdk.ConsAddress(oldPk.Address()), sdk.ConsAddress(newPk.Address())); err != nil {
 		return err
 	}
 
 	return k.Hooks().AfterConsensusPubKeyUpdate(ctx, oldPk, newPk, fee)
 }
 
-// setConsKeyToValidatorIdentifierMap adds an entry in the state with the current consKey to the initial consKey of the validator.
+// setConsAddrToValidatorIdentifierMap adds an entry in the state with the current consKey to the initial consKey of the validator.
 // it tries to find the oldPk if there is a entry already present in the state
-func (k Keeper) setConsKeyToValidatorIdentifierMap(ctx context.Context, oldPk, newPk sdk.ConsAddress) error {
-	pk, err := k.ConsKeyToValidatorIdentifierMap.Get(ctx, oldPk)
+func (k Keeper) setConsAddrToValidatorIdentifierMap(ctx context.Context, oldConsAddr, newConsAddr sdk.ConsAddress) error {
+	pk, err := k.ConsAddrToValidatorIdentifierMap.Get(ctx, oldConsAddr)
```

Comment:
Just to clarify, `pk` could be named to something like `previousConsAddr`?

Reply:
ty, changed it to validatorIdentifier which I think is more fitting